eos/b-decays/b-to-d-l-nu.cc

```diff
@@ -65,7 +75,8 @@ namespace eos
                 // only B_{d,u} mesons can decay in this channel
                 throw InternalError("BToDLeptonNeutrino: q = '" + o["q"] + "' is not a valid option for this decay channel");
             }
-
+            
+            //  e.g. [BCL2008], eq. (2), p. 1
```

Comment:
Please remove this comment since it is wrong. We can use any form factor below, but merely *default* BCL2008.